Resources/DoctrineMigrations/Version20180129102508.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
The migration script should be generated in the local-user-bundle (for mysql, pgsql and sqlite)

Reply:
https://github.com/ems-project/EMSLocalUserBundle/pull/2/files